sourcegraph/sourcegraph
# Insights/display-only-active-series-on-hover

Closes https://github.com/sourcegraph/sourcegraph/issues/36463

![PixelSnap 2022-06-07 at 17 41 30](https://user-images.githubusercontent.com/1855233/172495092-1c0a78da-c30f-45b9-bab4-52c9970165f7.png)

## Test plan

- Open storybook
- Go to "Smart Insights View Grid Example"
- Toggle any number of series on any insight
- Only those series should show in the tooltip on hover

## App preview:

- [Web](https://sg-web-insights-display-only-active.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-kiuxusimip.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
